parts/linux/cloud-init/artifacts/cse_start.sh

```diff
@@ -1,5 +1,7 @@
 CSE_STARTTIME=$(date)
 CSE_STARTTIME_FORMATTED=$(date +"%F %T.%3N")
+EVENTS_LOGGING_DIR=/var/log/azure/Microsoft.Azure.Extensions.CustomScript/events/
```

Comment:
did we ever determine from Chase if this was actually created by cse and if creating this manually had any implications?

Reply:
no chase said there should be nothing wrong with creating the directory manually, and I confirmed that I am still able to see the cse status in kusto with scriptless vhds.